gsl_projet/models.py

```diff
@@ -145,6 +145,13 @@ def to_notify(self):
             ),
         ).filter(dotations_count=F("programmation_count"), notified_at__isnull=True)
 
+    def with_at_least_one_programmated_dotation(self):
```

Comment:
```suggestion
    def with_at_least_one_programmed_dotation(self):
```

Bon de toute façon ici, utiliser programmation en anglais est je pense un faux ami et en anglais on dirait planned (programmation budgétaire = budget planning), et on est de toute façon dans la sauce car on a décidé d'avoir des noms de variables et objets en français mais d'action et de méthodes en anglais, mais en tout cas programmated ça n'existe pas !